lib/foreman_maintain/runner.rb

```diff
@@ -38,5 +43,23 @@ def ask_about_offered_steps(steps)
         @reporter.on_next_steps(self, steps)
       end
     end
+
+    def mock_skip_class(step)
+      Class.new(step.class.superclass) do
```

Comment:
We migth actually not need to show this, at least not for the case, where we offer fix procedure and the user chooses not to run it, this would cause complciations for multi-option case